Doc/source/coordinates.rst

```diff
@@ -1,32 +1,99 @@
+#############################################################################################
+spacepy.coordinates - Implementation of Coords class functions for coordinate transformations
+#############################################################################################
 
+.. contents:: Table of Contents
+    :depth: 2
+    :local:
 
-##############################################
-coordinates - module for coordinate transforms
-##############################################
 
-.. automodule:: spacepy.coordinates
+See also the `full API documentation <spacepy.coordinates>`.
 
-.. currentmodule:: spacepy.coordinates
+NOTES
```

Comment:
No need for all caps, and I don't think "notes" is a very informative heading. It can probably just be removed.

Reply:
This one is resolved